homeassistant/components/ziggo_next/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The Ziggo Next integration."""
+import asyncio
+import logging
+
+import voluptuous as vol
+from ziggonext import ZiggoNext
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONF_COUNTRY_CODE, DOMAIN, ZIGGO_API
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = ["media_player"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Ziggo Next component."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Ziggo Next from a config entry."""
+    api = ZiggoNext(
+        entry.data[CONF_USERNAME],
+        entry.data[CONF_PASSWORD],
+        entry.data[CONF_COUNTRY_CODE],
+    )
+    try:
+        api.initialize(_LOGGER, False)
```

Comment:
```suggestion
        await self.hass.async_add_executor_job(api.initialize,_LOGGER, False)
```
Is this doing I/O, if so it will need to be wrapped?